src/fmu/tools/rms/generate_petro_jobs_for_field_update.py

```diff
@@ -13,7 +13,6 @@
 import copy
 import json
 import pprint
-import warnings
 
 import yaml
 
```

Comment:
Will the imports here work? I think you need to import `roxar` and `rmsapi` instead of `_roxar` and `_rmsapi` ?
```python
try:
    import roxar  # type: ignore
    import roxar.jobs  # type: ignore

except ModuleNotFoundError:
    try:
        import rmsapi as roxar  # type: ignore
        import roxar.jobs  # type: ignore
```

You could potentially simplify by using the global `ROXAR` variable defined in `fmu.tools.__init__` . something like:
```python
if ROXAR:
    import rmsapi
    import rmsapi.jobs  # type: ignore
else:
    pass
```
and then use rmsapi in the script instead of roxar 🙂

Reply:
OK If we assume RMS version is new enough to use rmsapi, I will go for the last suggestion.

Reply:
The code in `fmu.tools.__init__` handles older RMS versions by importing `roxar` as `rmsapi,` so should work fine. But you could test inside RMS to make sure.

https://github.com/equinor/fmu-tools/blob/ab5c4c92f2398c7a0128662c655f06ce653408b2/src/fmu/tools/__init__.py#L10-L21

Reply:
Code is updated and should work for both RMS13.1.2 and RMS14.2. I also fixed a line in the test for generate_petro_jobs_for_field_update.py